### PR TITLE
Added KeyCDN

### PR DESCRIPTION
**Pull request by sigfriedseldeslachts**

### CODE REVIEW

**Comment by jvoisin** [comment]
Where can I check that this is working?

**Comment by sigfriedseldeslachts** [comment]
Check the response headers:
cdn.keycdn.com
36andmore-5b5e.kxcdn.com (will give SSL error as it is using ssl of CNAME)
...

If that is what you mean.

File: src/apps.json [hunk 1]
```diff
@@ -3914,6 +3914,16 @@
 			},
 			"website": "www.kentico.com"
 		},
+		"KeyCDN": {
+			"cats": [
+				31
+			],
+			"headers": {
+				"Server": "keycdn-engine"
```
Comment (jvoisin) [inline src/apps.json]: Please use `"^keycdn-engine$"` to speed up a bit the matching :)

**Comment by jvoisin** [comment]
Yup, I wanted to website to test your rule, thank you :)

**Comment by sigfriedseldeslachts** [comment]
Ok, I'll edit tomorrow.

**Comment by sigfriedseldeslachts** [comment]
I changed it.

**Comment by AliasIO** [comment]
Thanks!

**Pull request merged**